hack/create-account/BUILD.bazel

```diff
@@ -22,6 +22,5 @@ go_library(
 go_binary(
     name = "create-account",
     embed = [":go_default_library"],
-    pure = "on",
```

Comment:
Why do we need to delete this one?

Reply:
The PR is not ready. I'll let you know when it is ready.